go.mod

```diff
@@ -64,4 +64,6 @@ replace github.com/lightningnetwork/lnd/cert => ./cert
 
 replace git.schwanenlied.me/yawning/bsaes.git => github.com/Yawning/bsaes v0.0.0-20180720073208-c0276d75487e
 
+replace github.com/lightninglabs/neutrino => github.com/breez/neutrino v0.11.1-0.20200312075431-42a0bdf361ba
```

Comment:
We need to get rid of the replace statement now and point neutrino to master.

Reply:
neutrino dependency has been updated to master.